Release_Process.md

```diff
@@ -2,18 +2,19 @@
 
 This is a summary of the steps we go through to release a new version of BigchainDB Server.
 
-1. Update the `CHANGELOG.md` file
-2. Update the version numbers in `bigchaindb/version.py`. Note that we try to use [semantic versioning](http://semver.org/) (i.e. MAJOR.MINOR.PATCH)
-3. Go to the [bigchaindb/bigchaindb Releases page on GitHub](https://github.com/bigchaindb/bigchaindb/releases)
+1. Run `python docs/server/generate_schema_documentation.py` and commit the changes to docs/server/sources/schema.rst, if any.
+2. Update the `CHANGELOG.md` file
```

Comment:
We can use autonumbering with markdown, so each of these items can just be `1.`. Makes it much easier to diff.